--- ./php.ini-development.orig	2011-02-08 18:25:44.000000000 -0600
+++ ./php.ini-development	2011-04-01 22:42:50.000000000 -0500
@@ -938,6 +938,44 @@
 ; If you only provide the name of the extension, PHP will look for it in its
 ; default extension directory.
 ;
+
+; These are the extensions that are available for use on this PHP build.
+; If you do NOT need any of these extensions, you should comment them out.
+; That will probably increase both your security and performance.
+
+extension=bcmath.so
+extension=bz2.so
+extension=calendar.so
+extension=ctype.so
+extension=curl.so
+extension=dba.so
+extension=enchant.so
+extension=exif.so
+extension=ftp.so
+extension=gd.so
+extension=gettext.so
+extension=gmp.so
+extension=iconv.so
+extension=ldap.so
+extension=mbstring.so
+extension=mysql.so
+extension=mysqli.so
+extension=openssl.so
+extension=pdo.so
+extension=pdo_mysql.so
+extension=pdo_sqlite.so
+extension=pspell.so
+extension=shmop.so
+extension=snmp.so
+extension=soap.so
+extension=sqlite.so
+extension=sqlite3.so
+extension=tokenizer.so
+extension=wddx.so
+extension=xsl.so
+extension=zip.so
+extension=zlib.so
+
 ; Windows Extensions
 ; Note that ODBC support is built in, so no dll is needed for it.
 ; Note that many DLL files are located in the extensions/ (PHP 4) ext/ (PHP 5)
