diff -up fltk-1.1.10/configure.pkgconfig_xft fltk-1.1.10/configure
--- fltk-1.1.10/configure.pkgconfig_xft	2010-02-14 14:03:17.730616944 -0600
+++ fltk-1.1.10/configure	2010-02-14 14:05:05.369368026 -0600
@@ -12080,7 +12080,7 @@ if test "${ac_cv_lib_Xft_XftDrawCreate+s
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lXft  $LIBS"
+LIBS="`pkg-config --libs fontconfig xft` $LIBS"
 cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
@@ -12140,7 +12140,7 @@ if test $ac_cv_lib_Xft_XftDrawCreate = y
 #define USE_XFT 1
 _ACEOF
 
-			LIBS="-lXft $LIBS"
+			LIBS="`pkg-config fontconfig xft --libs` $LIBS"
 fi
 
 fi
