diff -U 3 -H -d -r -N -- openjpeg_v1_4_sources_r697/jp3d/libjp3dvm/CMakeLists.txt openjpeg_v1_4_sources_r697.new/jp3d/libjp3dvm/CMakeLists.txt
--- openjpeg_v1_4_sources_r697/jp3d/libjp3dvm/CMakeLists.txt	2011-01-02 19:14:46.000000000 +0100
+++ openjpeg_v1_4_sources_r697.new/jp3d/libjp3dvm/CMakeLists.txt	2011-04-10 13:23:06.102999992 +0200
@@ -36,4 +36,4 @@
   DESTINATION ${OPENJPEG_INSTALL_INCLUDE_DIR}/openjpeg3d-1.3
 )
 INSTALL(CODE
-  "EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -E create_symlink \$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/${OPENJPEG_INSTALL_INCLUDE_DIR}/openjpeg3d-1.3/openjpeg3d.h \$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/${OPENJPEG_INSTALL_INCLUDE_DIR}/openjpeg3d.h)")
+  "EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -E create_symlink openjpeg3d-1.3/openjpeg3d.h \$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/${OPENJPEG_INSTALL_INCLUDE_DIR}/openjpeg3d.h)")
diff -U 3 -H -d -r -N -- openjpeg_v1_4_sources_r697/libopenjpeg/CMakeLists.txt openjpeg_v1_4_sources_r697.new/libopenjpeg/CMakeLists.txt
--- openjpeg_v1_4_sources_r697/libopenjpeg/CMakeLists.txt	2011-01-02 19:14:45.000000000 +0100
+++ openjpeg_v1_4_sources_r697.new/libopenjpeg/CMakeLists.txt	2011-04-10 13:22:24.714999992 +0200
@@ -43,7 +43,7 @@
   DESTINATION ${OPENJPEG_INSTALL_INCLUDE_DIR}/${subdir} COMPONENT Headers
 )
 INSTALL(CODE
-  "EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -E create_symlink \$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/${OPENJPEG_INSTALL_INCLUDE_DIR}/${subdir}/openjpeg.h \$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/${OPENJPEG_INSTALL_INCLUDE_DIR}/openjpeg.h)")
+  "EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -E create_symlink ${subdir}/openjpeg.h \$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/${OPENJPEG_INSTALL_INCLUDE_DIR}/openjpeg.h)")
 
 # install man page of the library
 INSTALL(
